src/dev/run_i18n_check.ts

```diff
@@ -105,16 +110,17 @@ run(
         {
           title: 'Validating Default Messages',
           skip: skipOnNoTranslations,
-          task: ({ config }) =>
-            new Listr(extractDefaultMessages(config, srcPaths), { exitOnError: true }),
+          task: ({ config }) => {
+            return new Listr(extractDefaultMessages(config!, srcPaths), { exitOnError: true });
+          },
         },
         {
           title: 'Compatibility Checks',
           skip: skipOnNoTranslations,
-          task: ({ config }) =>
+          task: ({ config }) => {
             new Listr(
               checkCompatibility(
-                config,
+                config!,
```

Comment:
Same here, you dont need this change.

Reply:
Yes, inside the function I did a check for the presence of a config. but there was still an error, although I did the same thing in another function, so I set it to ignore, since I am sure that there will be an error without the config. I will try to fix it somehow, if I succeed

Reply:
Wouldn't it be more appropriate to check for config when called? i.e. if(config){}
otherwise the function itself will have to describe config: i18nConfig | undefined.

Reply:
Yeah, i leave that to you. The only thing i'm really looking for is a consistent type for the context so that i makes readability easier. So i like the optional type for `config` and if you need to add a check for config in the helper functions to get rid of the error, thats okay by me :)